src/components/layout.js

```diff
@@ -5,11 +5,11 @@
  * See: https://www.gatsbyjs.org/docs/use-static-query/
  */
 
-import React from "react"
-import PropTypes from "prop-types"
-import { useStaticQuery, graphql } from "gatsby"
+import React from 'react'
```

Comment:
this is weird, i think some auto-linting/code formatting happened. i'd just ignore it for this PR